content/docs/ref/publish.md

```diff
@@ -4,7 +4,8 @@
 cml publish [options] <image file>
 ```
 
-Publicly host an image for displaying in a CML report.
+Publicly host an image for displaying in a CML report. Used internally by
```

Comment:
I think it should be documented that the report is parsed for any links to local images and those images are then uploaded to public storage.

Reply:
That's in `/doc/ref/comment#options` not here :)